modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/services/liferay/graphql/queries.js

```diff
@@ -30,23 +30,9 @@ export const getDXPCDataCenterRegions = gql`
 `;
 
 export const getKoroneikiAccounts = gql`
-	query koroneikiAccounts(
-		$aggregation: [String]
-		$filter: String
-		$page: Int = 1
-		$pageSize: Int = 20
-		$search: String
-		$sort: String
-	) {
+	query getKoroneikiAccounts($filter: String) {
```

Comment:
I think would be better to maintain the older implementation, since all the parameters are optional, should not have any problem.

Reply:
I made this because there are other queries with the same optional params and thought it that we could growth it step-by-step. It's does make sense?

Reply:
Makes sense Jair, in the future, let's use all the optional variables here, will be better for us to have a big picture from what is expected by the query or mutation